templates/documents/general_view.html

```diff
@@ -309,8 +309,43 @@ <h4>{xl t='Contents'|escape:'html'}</h4>
 			 $file->get_mimetype() eq "image/gif" or
 			 $file->get_mimetype() eq "application/pdf" }
 			<iframe frameborder="0" type="{$file->get_mimetype()}" src="{$web_path}as_file=false"></iframe>
-			{elseif $file->get_ccr_type($file->get_id()) ne "CCR" and
-       $file->get_ccr_type($file->get_id()) ne "CCD"}
+            {elseif $file->get_mimetype() eq "application/dicom"}
+            <!-- DWV -->
+            <div id="dwv" src="{$web_path}as_file=false">
+                <!-- Toolbar -->
+                <div class="toolbar"></div>
+                <div class="warn_diagnostic">Not For Diagnostic Use</div>
```

Comment:
Lets translate this:
``````
<div class="warn_diagnostic">{xl t='Not For Diagnostic Use'|escape:'html'}</div>
``````